package.json

```diff
@@ -9,7 +9,7 @@
     "build": "gulp build",
     "build:client": "gulp build:client",
     "clean": "gulp clean",
-    "dev": "gulp build:dev && env-cmd -f ./.env gulp serve:dev",
+    "dev": "gulp build:dev && env-cmd -f ./.env.dev gulp serve:dev",
```

Comment:
I think I still like keeping `.env` since it's not tracked by version control (or the container images)